etc/app-config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "modules": [
+    {
+      "executable_path": "/usr/local/bin/viam-csi",
+      "name": "csi_cam_module"
+    }
+  ],
+  "components": [
+    {
+      "model": "viam:camera:csi",
+      "attributes": {
```

Comment:
What if there are multiple cameras plugged in? How will the resource know which one to connect to?

Reply:
Ah, I see we do let you specify video_path. Can you add this to the sample config?

Reply:
Yep, added to sample config.